src/homepage.js

```diff
@@ -0,0 +1,21 @@
+function homepage () {
+    const content = document.getElementById("content");
+    const pageContent = document.createElement("div");
+    pageContent.setAttribute("id", "pageContent");
+    content.appendChild(pageContent);
+
+    const heading = document.createElement("h1");
+    heading.textContent = "Restaurant Homepage";
+    pageContent.appendChild(heading);
+
+    const hours = document.createElement("h2");
+    hours.textContent = "Opening Hours";
+    pageContent.appendChild(hours);
+
+    const hours2 = document.createElement("div");
```

Comment:
`hours2` is not a very good name, can name it something better.

Reply:
Could do something like `operatingHours`